modules/nf-core/odgi/squeeze/main.nf

```diff
@@ -0,0 +1,39 @@
+process ODGI_SQUEEZE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::odgi=0.8.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/odgi:0.8.2--py39h2add14b_0':
+        'quay.io/biocontainers/odgi:0.8.2--py39h2add14b_0' }"
+
+    input:
+    tuple val(meta), path(graphs)
+
+    output:
+    tuple val(meta), path("*.og"), emit: graph
```

Comment:
Can we align the emits?

Reply:
Sorry, what is it you want aligned? It is only one emit.

Reply:
Ah, it was not visible in the code shown here. Done.